app/services.py

```
@@ -173,5 +190,89 @@ def delete_chat_session(self, member_id: int, session_id: str) -> None:
+    normalized = message.strip()
+    for name, product_key in PRODUCT_NAME_TO_KEY.items():
+        if name in normalized:
+            return product_key
+    return None
+
+
+async def fetch_loan_eligibility(
+    access_token: str,
+    product_key: str,
+) -> LoanEligibilityResponse:
+    async with httpx.AsyncClient(timeout=5.0) as client:
```

Ruff (0.15.9)

[error] Undefined name `httpx`

(F821)